src/formatDate.js

```diff
@@ -8,7 +8,61 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const FROM_SEPARATOR_FORMAT = fromFormat[3];
+  const TO_SEPARATOR_FORMAT = toFormat[3];
+  let year, month, day;
+  const RESULT = [];
```

Comment:
The 'year', 'month', 'day', and 'RESULT' variables can be declared using 'let' instead of 'const' since 'RESULT' is being modified later on in the code. Additionally, consider using lowerCamelCase for variable names as per JavaScript naming conventions.